_docs/setup-collator-node.md

```diff
@@ -44,62 +39,85 @@ The Turing Network is live on Kusama. If you're interested in collating, please
 - [Docker image repository](https://hub.docker.com/repository/docker/oaknetwork/turing)
 - [Turing Network parachain chain spec](https://github.com/OAK-Foundation/OAK-blockchain/blob/master/node/res/turing.json)
 - [Kusama chain spec](https://github.com/paritytech/polkadot/blob/master/node/service/res/kusama.json)
-- [Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d)
+- [Turing Network Telemetry](https://telemetry.polkadot.io/#list/0x0f62b701fb12d02237a33b84818c11f621653d2b1614c777973babf4652b535d)
+
+## Set up a collator
 
-## How to setup your node
+### Step 1 - Create a wallet for your collator
 
-### Step 1: Get your binary ready
+In this step, we'll generate a node key for your collator. The simplest method to accomplish this is by using Substrate's [Subkey command-line tool](https://docs.substrate.io/reference/command-line-tools/subkey/). After you've installed the tool, execute the command subkey `generate-node-key`. This command will yield two lines of output. The first line represents a new Substrate wallet address, and the second line is its corresponding private key.
 
-#### Option 1: Grab a compiled binary from OAK's Github
-If you are using Ubuntu (20.04+ LTS x64), you can run the binary compiled by OAK that can be found [here](https://github.com/OAK-Foundation/OAK-blockchain/releases/latest). You'll use this to run your collator on your node. To acquire a copy of this via command line, use the commands below.
+### Step 2 - Start up the collator program
+
+#### Option 1: Download a release binary(Recommended)
```

Comment:
The major changes are here,
1. I made sure all the steps for each option in one place instead of being splitted up in multiple steps.
2. Updated the set up and upgrade command to use .deb file for installation.